src/pkg/cli/client/byoc/aws/byoc_integration_test.go

```diff
@@ -15,7 +16,12 @@ import (
 var ctx = context.Background()
 
 func TestDeploy(t *testing.T) {
-	b := NewByocProvider(ctx, client.GrpcClient{}, "ten ant") // no domain
+	b, err := NewByocProvider(ctx, client.GrpcClient{}, "ten ant") // no domain
+	if err != nil {
```

Comment:
since this file is tagged as `integration`, we should expect credentials to be available so we should ensure we get `nil`

Reply:
right, we will do t.Skip() if the error is an "unauthorized" error, and t.Fatal() when it is anything other than that.